resources/run_adf.sh

```diff
@@ -307,14 +361,16 @@ if [[ $? -ne 0 ]]; then
         fi
         cd $MONGOHOUSE_DIR
 
-        # for now, we checkout a specific working commit of ADF
-        # TODO SQL-1374: Find a better long term solution
-        git checkout 247a246
 
         export GOPRIVATE=github.com/10gen
+        # make sure mod vendor is cleaned up
+        $GO mod vendor
```

Comment:
We haven't had to add this vendoring step before.  Do you have this set in your env, something like `export GOFLAGS=-mod=vendor`?  If so, try unsetting that and see if run_adf works without this vendoring.

Reply:
This is incompatible with the setup instructions for [sqlproxy](https://github.com/10gen/sqlproxy#unix), so it seems easy enough (and works) to just run it as part of the script.